test/predictions/repo_test.exs

```diff
@@ -58,6 +59,67 @@ defmodule Predictions.RepoTest do
       end
     end
 
+    test "filtes out predictions with no departure" do
+      five_minutes_in_future = DateTime.add(Timex.now(), 5, :minute)
+
+      {:ok, five_minutes_in_future_string} =
+        Timex.format(five_minutes_in_future, "{ISO:Extended:Z}")
+
+      expect(MBTA.Api.Mock, :get_json, fn _, _ ->
+        %JsonApi{
+          data: [
+            %JsonApi.Item{
+              attributes: %{
+                "departure_time" => nil,
+                "arrival_time" => five_minutes_in_future_string,
+                "direction_id" => 1
+              },
+              relationships: %{
+                "route" => [
+                  %{
+                    id: "Red"
+                  }
+                ],
+                "trip" => [],
+                "vehicle" => [],
+                "stop" => [
+                  %{id: "StopID"}
+                ]
+              }
+            },
+            %JsonApi.Item{
+              attributes: %{
+                "departure_time" => five_minutes_in_future_string,
+                "arrival_time" => five_minutes_in_future_string,
+                "direction_id" => 1
+              },
+              relationships: %{
+                "route" => [
+                  %{
+                    id: "Red"
+                  }
+                ],
+                "trip" => [],
+                "vehicle" => [],
+                "stop" => [
+                  %{id: "StopID"}
+                ]
+              }
+            }
+          ]
+        }
```

Comment:
Are all of these attributes necessary for exercising the test? Would it work just setting the departure_time? These two structs are exactly the same except for that one attribute, so there should be a shorter and clearer way to communicate that.

Reply:
Yes, all these fields are required.  The parsing errors out if any of these fields are missing.  Ideally I'd like to mock away some of that, but it seemed outside the scope of this fix.